plugins/rapid7_insight_agent/help.md

```diff
@@ -80,19 +80,22 @@ Example output:
 
 #### Get Agent Details
 
-This action is used to find and display detailed information about a device
+This action is used to find and display detailed information about a device. If additional pages of agents are 
+available, the action should be run again with the returned next cursor
```

Comment:
the formatting of this seems a bit strange and its also not the same as the other one with a similar second sentence